LDAModel.py

```diff
@@ -0,0 +1,289 @@
+from ClusteringModel import ClusteringModel
+from octis.models.model import AbstractModel
+import numpy as np
+from gensim.models import ldamodel
+import gensim.corpora as corpora
+import octis.configuration.citations as citations
+import octis.configuration.defaults as defaults
+from gensim.models import TfidfModel
+
+
+class LDAModel(ClusteringModel, AbstractModel):
+    id2word = None
```

Comment:
These are class instance variables but seem to be used as instance variables. If they really are class instance variables, remove the code that sets the values within instance methods.